compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/input/key/NativePointerEvent.kt

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.input.key
+
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.input.pointer.PointerEventType
+import androidx.compose.ui.input.pointer.PointerType
+
+expect class PlatformPointerEvent
+data class NativePointerEvent(
```

Comment:
data class cannot be used as public API. See https://kotlinlang.org/docs/jvm-api-guidelines-backward-compatibility.html#don-t-use-data-classes-in-an-api

Reply:
but, but
<img width="1354" alt="Screenshot 2024-04-03 at 12 04 37" src="https://github.com/JetBrains/compose-multiplatform-core/assets/25667761/26ba00b7-2cd0-45a3-bd5d-f8c534d339c1">

so, ok, the goal of this PR was to move entities from SKIKO that de-facto were a) data classes and b) public API
I'm too concerned to change the far they are data classes  in this particular PR  which is about mechanical shift from changing the location of some entities from skiko to compose - I just afraid that somewhere in code there's some reliability on the fact that this is exactly data class, may be the real answer is to make this API well, not public.
@igordmn what do you think?

Reply:
The initial purpose of this API was to abstract away of platform details inside Skiko.

Compose iteself already have such API (KeyEvent, PointerEvent), so if we move similar API from Skiko, we need to move it as internal API.

Reply:
As we decided to remove Skiko API, it should be removed, not moved to Compose

Reply:
We still have new public API:
- PlatformKeyboardEvent
- PlatformPointerEvent
- NativeKeyEvent
- InputModifiers

NativeKeyEvent is needed because it is actual of common API, but by design it should point to real native event (NSEvent, KeyboardEvent)

Reply:
> NativeKeyEvent is not ours, we are only actualising it

Yes, I mentioned this in the last sentence. We need it public, but we shouldn't introduce a new type for it. As discussed privately, it should either point to a real native event, or to `Any` + something like `toWebEventOrNull()`

Reply:
I vote for Any, and moving this to skikoMain from desktopMain:
```
actual typealias NativeKeyEvent = Any
```